### PR TITLE
Convert lisk-passphrase tests to jest - Closes #4682

### LINKED ISSUE #4682: Convert lisk-passphrase mocha test to jest
### Description

Convert the tests in `lisk-passphrase` library from `mocha` to `jest`.

### Motivation

Consistent use of `jest` testing.

### Acceptance Criteria

`jest`

### PR DESCRIPTION
**Pull request by valamidev**
### What was the problem?
Tests were in mocha.

### How did I solve it?
Tests converted to jest.

### How to manually test it?
`npm run test`

### Review checklist

- [x] The PR resolves #4682
- [x] All new code is covered with unit tests
- [x] Relevant integration / functional tests are added
- [x] Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
- [x] Documentation has been added/updated

### CODE REVIEW

**Comment by valamidev** [comment]
> `elements/lisk-passphrase/test/tsconfig.json` can be removed and `elements/lisk-passphrase/jest.config.js` can symlink?
> Also, `elements/lisk-passphrase/tsconfig.json` needs to be updated?

Jest config template cannot be used yet, we need to change test filenames first to match the matcher condition. 

`testMatch: ['<rootDir>/test/**/?(*.)+(spec|test).+(ts|tsx|js)']`

**Comment by shuse2** [comment]
@valamidev I think test file name should be updated to `xzxx.spec.ts`

File: elements/lisk-passphrase/test/index.spec.ts [hunk 1]
```diff
@@ -12,23 +12,22 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import { expect } from 'chai';
 import * as passphrase from '../src';
 
 describe('passphrase index.js', () => {
 	it('should export an object', () => {
-		return expect(passphrase).to.be.an('object');
+		return expect(passphrase).toEqual(expect.any(Object));
```
Comment (mitsuaki-u) [inline elements/lisk-passphrase/test/index.spec.ts]: `.toBeObject()`?

Comment (valamidev) [reply elements/lisk-passphrase/test/index.spec.ts]: Could be, but `.toBeObject()` is from `jest-extended`. I prefer to use the basic jest matchers and assertions where it is possible.

**Pull request merged**